### PR TITLE
Require Ruby 3.0 and 2.7

### LINKED ISSUE #78: New ruby syntax support
Hi there,

I'm trying to use querly with a project on ruby 2.7 and it causes a couple of errors like 
```sh
#<Parser::SyntaxError: unexpected token tDOT3>
#<Parser::SyntaxError: unexpected token tRBRACK>
```

It seems to be a problem of using `Parser::Ruby25` class instead of `Parser::CurrentRuby`.

### PR DESCRIPTION
**Pull request by ybiquitous**
This change aims to support newer Rubies 3.0 and 2.7.
Also, this requires Ruby 2.7 or newer (**breaking change**).

Close #78

### CODE REVIEW

File: .github/workflows/ruby.yml [hunk 1]
```diff
@@ -8,15 +8,15 @@ on:
```
Comment (ybiquitous) [inline .github/workflows/ruby.yml:8]: [note] This file change is the almost same as #87.

**Comment by ybiquitous** [comment]
@soutaro Could you review this PR, please?

**Review by soutaro — APPROVED** [review]
🙏

**Comment by soutaro** [comment]
@ybiquitous I'm really sorry for the delay. New version 1.3.0 is out.

**Comment by ybiquitous** [comment]
@soutaro Thank you so much! 😊

**Pull request merged**